database/README.md

```diff
@@ -274,6 +275,23 @@ Column                    | Type                         | Nullable | Details
 Indexes:
 - PRIMARY KEY: btree(`uid`)
 
+### order\_quote\_interactions
+
+Solver responding to quote request provides a list of interactions which need to be executed to fulfill the quote. When order is created
+these interactions are copied from quote. These interactions are stored persistently and can be used to audit auction winning order.
```

Comment:
This should be treated like user facing documentation and should therefore use proper grammar.

Reply:
Updated.